### ISSUE TITLE
Go live (without Varnish)

### ISSUE THREAD

**Issue by stevenday** [mysociety/mapit.mysociety.org#19]
I'm going to attempt to put this code live in place of mapit.mysociety.org, without the varnish config in place initially, as a test of things. 
## Go live plan
- [x] update vhosts.pl to swap the site definition used for mapit from `mapit` to `mapit.mysociety.org` (don't commit it, so it's easy to rollback later) 
- [x] update vhosts/mapit.mysociety.org_conf_general.yml.ugly et al to have the necessary config for the live site (maybe worth moving this to a folder/individual site setup?)
- [x] delete the `django_site` table from the db (to avoid a migration issue later)
- [x] back up the mapit db
### On stork (for argument's sake)
- [x] backup `/data/vhosts/mapit.mysociety.org`
- [x] delete `/data/vhosts/mapit.mysociety.org` (seems safer to make sure everything is re-created afresh, there are no weird git conflict issues, and nothing old is left over inadvertently)
- [x] run `mysociety vhost` to deploy the site
- [x] Double check that vhost isn't doing anything terrible
### On milo
- [x] Repeat the backup `/data/vhosts/mapit.mysociety.org`, delete, deploy steps from stork
## Roll back plan

If something unexpected goes wrong, the rollback plan is as follows:
- [ ] Reset the vhosts.pl site setting for mapit.mysociety.org
- [ ] Restore the db from before the changes
- [ ] Restore the `/data/vhosts/mapit.mysociety.org` folder
- [ ] Re-deploy mapit (probably could just graceful apache, but better safe than sorry).

**Comment by stevenday** [comment]
@dracos - can you see anything obvious I've missed, or any way to do this better?

**Comment by dracos** [comment]
Looks good to me. There are some changes to upstream that need including here first (e.g. sass compilation now using libsass), to make sure that all deploys okay (that should have shown in testing though, hopefully?)

mapit is already running on Django 1.8, so I guess by the sites thing you mean because the table is present from a time before we removed sites in mapit – I guess the table could have been removed at that point (or now first off?), and then it would be recreated by the app appearing again?

Updating vhosts.pl can be done on stork too without committing, to make it easier to roll back if something does go wrong. Not sure the config file is that long ;) And the mapit db is already backed up by the backup process :)

**Comment by stevenday** [comment]
@dracos - thanks for the reminder about the libsass update - I've updated the script in this repo to match (scripts in bin aren't available from mapit when we install it via pip so I had to copy it). I've also mirrored the conf/packages updates. I can't see anything else in the recent commits that should affect things.

With the sites table, yes, you're right - the sites framework was removed in https://github.com/mysociety/mapit/commit/4b7e5f8463d659c6cbe79f942d40f56b8c93a77b, so the table that's currently in the db must have been from then. I'll remove it now to solve that conflict, as it makes things simpler.

Thanks!
